renters_rights/units/contactus.py

```diff
@@ -0,0 +1,9 @@
+rom django import forms
+
+class NameForm(forms.form):
+    your_email = forms.CharField(label = 'Your Email', max_length = 100)
+    your_name = forms.CharField(label = 'Your Name', max_length = 100)
+    your_comments = forms.CharField(label = 'Your Comments', max_length = 100)
```

Comment:
You'll want to add `widget=forms.Textarea` to this field so it renders as a multiline text area.
And let's make the max_length 10,000.